go/engine/device_keygen.go

```diff
@@ -211,6 +215,9 @@ func (e *DeviceKeygen) setup(m libkb.MetaContext) {
 	}
 
```

Comment:
let's add something like
```
if m.G().Env.GetRunMode() != libkb.DevelRunMode && (e.args.naclSigningKeyPair != nil || e.args.naclEncryptionKeyPair != nil) {
    e.runErr = errors.Make("Trying to pass key pair agument to device keygen")
    return 
}
```
in case someone misunderstands and tries to utilize this in prod